planning/behavior_path_planner/autoware_behavior_path_avoidance_by_lane_change_module/src/scene.cpp

```diff
@@ -275,14 +275,10 @@ double AvoidanceByLaneChange::calcMinAvoidanceLength(const ObjectData & nearest_
 
 double AvoidanceByLaneChange::calcMinimumLaneChangeLength() const
```

Comment:
Maybe better to rename to differentiate between single lc length and entire dist buffer

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8849/commits/9a0f342290328d85911745afc1e7757a8f8ff7b1